packages/db/fauna/resources/Function/findMetrics.js

```diff
@@ -0,0 +1,82 @@
+import fauna from 'faunadb'
+
+const {
+  Function,
+  CreateFunction,
+  Query,
+  Lambda,
+  Count,
+  Exists,
+  If,
+  Update,
+  Sum,
+  Match,
+  Index,
+  Map,
+  Get,
+  Var,
+  Select,
+  Documents,
+  Collection,
+  Paginate
+} = fauna
+
+const name = 'findMetrics'
+const body = Query(
+  Lambda(
+    [],
+    {
+      data: {
+        usersTotal: Count(Documents(Collection('User'))),
+        uploadsTotal: Count(Documents(Collection('Upload'))),
+        contentTotal: Count(Documents(Collection('Content'))),
+        contentTotalBytes: Sum(Match(Index('content_sizes'))),
+        dealsActiveTotal: Count(Match(
+          Index('deals_by_status'),
+          'Active'
+        )),
+        dealsQueuedTotal: Count(Match(
+          Index('deals_by_status'),
+          'Queued'
+        )),
```

Comment:
I did not think the number of active or queued deals was interesting for NFT.Storage and no-one has asked for it yet! The metric we have in NFT.storage is the number of uploads that are in queued/active deals.

Reply:
dropping for now

Reply:
A metric that gets referred to a lot is % of successful deals. We should try and add that. Can be a subsequent PR.

Reply:
Yes, I had the total deals metric. But Riba might also give us this information from his side, so we might not even need to track it. @alanshaw suggested we wait for now